CorosHeader header
int32 message_code
int32 robot_id
int32 task_id
float64 cost

